src/pocketmine/lang/Installer/deu.ini

```diff
@@ -0,0 +1,51 @@
+language_has_been_selected = Deutsch wurde erfolgreich als Sprache ausgewählt.
+skip_installer = Möchtest du den Einrichtungsassistenten überspringen?
+
+welcome_to_pocketmine = Willkommen bei Genisys!\nBevor es mit der Einrichtung deines neuen Servers losgehen kann, musst du die Lizenz akzeptieren.\nGenisys ist unter der LGPL Lizenz lizenziert, welche du in der Datei LICENCE in diesem Ordner nachlesen kannst.
+accept_license = Akzeptierst du die Lizenz?
+you_have_to_accept_the_license = Um Genisys weiterverwenden zu können, musst du der LGPL Lizenz zustimmen.
```

Comment:
Okay, if I have understood the Genisys members correctly then we should change it to GPL v3 and later. This also means that we have to do it in all the other language files of the Installer too.